packages/common-types/src/index.ts

```diff
@@ -1,10 +1,43 @@
 /* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
  * Apache-2.0 */
 
+import { IItem } from "@esri/arcgis-rest-common-types";
+
 /**
  * One small step for Hub, one large leap for hubkind
  */
-export interface IHasZM {
-  hasZ?: boolean;
-  hasM?: boolean;
-}
\ No newline at end of file
+export interface IInitiative {
+  item?: IInitiativeItem;
+  data?: IInitiativeData;
+}
+
+export interface IInitiativeItem extends IItem {
+  type: "Hub Initiative";
+}
+
+export interface IInitiativeData {
+  assets: [any];
+  source: string;
+  values: {
+    collaborationGroupId: string;
+    openDataGroupId: string;
+    followerGroups: any;
+    steps: [string];
+    bannerImage: any;
+    website: IHaveNoIdeaWhatToCallThis;
+    informTools: IHaveNoIdeaWhatToCallThis;
+    listenTools: IHaveNoIdeaWhatToCallThis;
+    monitorTools: IHaveNoIdeaWhatToCallThis;
+    initiativeKey: string;
+  };
+}
+
+export interface IHaveNoIdeaWhatToCallThis {
```

Comment:
I'mma guess `IInitiativeTools`, but I'm not even sure that these are used anymore. Let's make sure we have a recent, valid initiative first and that we need to get into this level of detail.